apps/frontend/.env.example

```
@@ -7,4 +7,12 @@ NEXT_PUBLIC_FIREBASE_APP_ID=#1:736891312580:web:2752bc815204c69fcbec91
 
 NEXT_PUBLIC_BACKEND_URL=#http://localhost:3000
 NEXT_PUBLIC_HASURA_GRAPHQL_URL=#http://localhost:8080/v1/graphql
-NEXT_PUBLIC_HASURA_ADMIN_SECRET=#myadminsecretkey
\ No newline at end of file
+NEXT_PUBLIC_HASURA_ADMIN_SECRET=#myadminsecretkey
+
+# TrustlessWork (server-side only — do NOT prefix with NEXT_PUBLIC)
+TRUSTLESS_WORK_API_URL=https://dev.api.trustlesswork.com
+TRUSTLESS_WORK_API_KEY=your_trustlesswork_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRUSTLESS_WORK_API_KEY key should go before the TRUSTLESS_WORK_API_URL key

(UnorderedKey)